### ISSUE TITLE
UX Suggestion

### ISSUE THREAD

**Issue by pieceofduke** [lazerwalker/clojurescript-koans#32]
When user hits return after entering correct answer, he's transferred to next page immediately.
Better leave him on the same page with some indication of correctness, to contemplate the correct answer.
This may seem unimportant, but it's not: such a change will improve learning significantly.
For more info, google principles of programmed instruction (PI).
I hope it's trivial to implement.

**Comment by lazerwalker** [comment]
That makes a lot of sense to me; thanks for the great suggestion!

Pull requests gladly accepted ;)

**Comment by pieceofduke** [comment]
I'm not that good at coding yet )

**Comment by LeaveNhA** [comment]
@pieceofduke wanna be good at it ?

**Comment by LeaveNhA** [comment]
Actually, I can implement but there are seven PRs in the queue and I just want to know, @lazerwalker walk out of this project ?

**Comment by mkremins** [comment]
Yeah, this project isn't really maintained anymore. Em stepped back to focus on other things in 2016 or 2017, at which point I became the main maintainer. For the last few years I haven't really had time to work on it myself, so it's kinda been languishing. That said, I'd be happy to screen a PR for this issue if you decide to implement a fix!

**Comment by LeaveNhA** [comment]
Sad to hear that. This project is my left hand while I'm teaching Clojurescript to my students.
Of course, I can implement a fix. I am writing this to my schedule. Let's see what I can do.
